src/renderer/src/app.tsx

```diff
@@ -1,25 +1,42 @@
 import { createRoot } from 'react-dom/client';
 import Navbar from './components/navbar';
 import RadioContainer from './components/radio/radio-container';
-import Sidebar from './components/sidebar/sidebar';
 
 import ErrorDialog from './components/error';
 import Bootstrap from './components/bootstrap';
 import Mini from './components/mini';
 
 import './index.scss';
 import './style/app.scss';
+import FocusBar from './components/focusBar';
+// import { useState } from 'react';
+// import Updater from './components/updater/Updater';
 
 function App() {
+  // const [updateCheckDone, setUpdateCheckDone] = useState(false);
+
+  // if (!updateCheckDone) {
+  //   return (
+  //     <Updater
+  //       onFinish={() => {
+  //         setUpdateCheckDone(true);
+  //       }}
+  //     />
+  //   );
+  // }
+
```

Comment:
Did you mean to leave these in commented out?

Reply:
Yep, Electron Builder is being ridiculous at the moment with update handling, we need to take a proper look into this and I think it maybe outside the scope of this PR. Therefore, I think leave the code there as it will get used in the future and is part of the design change but yeh I had a few issues on Windows where the application was still in use when trying to quit and install the new update and on macOS it just wouldn't quit the app and I couldn't be bothered faffing about with electron builder so maybe we take a look at this at a later date.

Reply:
Or take it out and add it back in when the update stuff is done in a separate PR ;)

(Sorry, I just get 🤢 when I see commented out stuff making it into a main branch 😊)

Reply:
(Also, when is electron-builder NOT ridiculous? 😂)